test/test_binary_ufuncs.py

```diff
@@ -335,15 +337,38 @@ def test_div_rounding_nonfinite(self, device, dtype):
                              exact_device=False, exact_dtype=exact_dtype)
 
         # Compare contiguous (likely vectorized) against non-contiguous (not vectorized)
-        storage = torch.empty((20, 20), dtype=dtype, device=device)
-        storage[::2, ::2] = a
-        storage[1::2, 1::2] = b
+        a_noncontig = torch.empty([2 * i for i in a.shape], dtype=dtype, device=device)[::2, ::2]
```

Comment:
Do we really want empty tensors here?

Also make_tensor now has a `noncontiguous` kwarg to create noncontiguous tensors

Reply:
> Do we really want empty tensors here?

Yes, it gets filled on the next line.

> Also make_tensor now has a noncontiguous kwarg to create noncontiguous tensors

I don't want random data, I specifically want `a` and `b` copied into a non-contiguous tensor.